package.json

```diff
@@ -18,39 +18,21 @@
     "lint": "eslint . --ignore-path .gitignore",
     "lint:ci": "npm run lint -- -o lint-results.json -f json",
     "danger": "./bin/duti",
-    "duti:fix": "npm run lint -- --fix && npm run fmt",
-    "fmt": "prettier --write {*.js,{src,bin}/**/*.js} --no-semi --single-quote --trailing-comma es5"
-  },
-  "prettier": {
-    "singleQuote": true,
-    "semi": false,
-    "trailingComma": "es5"
-  },
-  "eslintConfig": {
-    "extends": "smartprocure",
-    "parserOptions": {
-      "ecmaVersion": 8,
-      "sourceType": "module"
-    },
-    "env": {
-      "node": true
-    }
+    "duti:fix": "npm run lint -- --fix"
   },
   "dependencies": {
     "bluebird": "^3.5.0",
     "cosmiconfig": "^2.2.2",
-    "futil": "^1.54.0",
-    "lodash": "^4.17.11",
+    "futil": "1.69.0",
+    "lodash": "4.17.21",
```

Comment:
Does it make sense to make these peer dependencies?  I think every repo that uses has them as a dep and that way were not adding multiple versions to the dep tree?

Reply:
actually this is always a dev dependency, probably not worth the effort.